composer.json

```diff
@@ -47,7 +47,7 @@
             "Tests\\": "tests/"
         }
     },
-    "minimum-stability": "stable",
+    "minimum-stability": "RC",
```

Comment:
This is here to use composer:1.10.0-rc, right ? which allow symfony/console:5.0 ?

Reply:
Yes